README.md

```
@@ -112,6 +112,12 @@ What to expect
 - Queries use the cache when available; otherwise scan the journal.
 - Everything is just Git — no external servers, no hidden DBs.
 
+### Safety Guard
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)